lodash/lodash.d.ts

```diff
@@ -263,12 +263,12 @@ declare module _ {
         * @param array Array to compact.
         * @return (Array) Returns a new array of filtered values.
         **/
-        compact<T>(array: Array<T>): T[];
+        compact<T>(array?: Array<T>): T[];
```

Comment:
https://lodash.com/docs#compact
I think `compact` is not allowed 1st argument to be optional.